.gitignore

```diff
@@ -0,0 +1,2 @@
+private-values.yaml
```

Comment:
Curious to hear others thoughts on if we should have this ignore all private pre-fixed items? 

I believe it is:
```suggestion
private-*
```

Reply:
wdyt @johnmcollier @coreydaley @thepetk ? ^^^

Reply:
If we are going to include a glob of files, i would rather exclude a directory that can be used instead, otherwise I think the more precise one is better.